src/components/ui/BaseSort.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="lg:flex p-1 mt-4 sm:mx-3 mb-5 justify-between">
+    <div class="flex my-3 p-1 lg:h-10 sm:h-9">
+      <BaseButton
+        :class="{ 'bg-emerald-400 border-none': isActiveButton('title') }"
```

Comment:
create a function which will return 'bg-emerald-400 border-none' oif the condition is true

Reply:
Also you should create an array of object with the BaseButton properties and v-for on that, so you dont have to write 4 separate BaseButtons